lib/factory.ts

```diff
@@ -5,8 +5,10 @@ export interface AnyFactories {
   [key: string]: Factory<any>;
 }
 
+const SEQUENCE_START_VALUE: number = 0;
```

Comment:
I wonder if the sequence should start at 1. I wrote the original code for this, and I don't think it was intentional on my part that it start at 0.

Reply:
Updated to starting value to `1`